addons/laserpointer/XEH_preInit.sqf

```diff
@@ -6,4 +6,11 @@ PREP_RECOMPILE_START;
 #include "XEH_PREP.hpp"
 PREP_RECOMPILE_END;
 
+["visionMode", {
```

Comment:
Isn't this negligible either way? I'd hate to use `removeEventhandler` for this.